python/foglamp/common/storage_client/payload_builder.py

```diff
@@ -604,8 +604,45 @@ def TIMEBUCKET(cls, timestamp, size="1", fmt=None, alias=None):
 
         return cls
 
+    @staticmethod
+    def filter_identifier(identifier):
+        # Characters `(backtick),
+        #            '(single quote),
+        #            "(double quote),
+        #            [(left square bracket),
+        #            ](right square bracket)
+        # are not allowed
```

Comment:
this should not filter but disallow! AND see where escaping is needed.

I think we are handling only backtick and single quote. We need more like `colon`, 'semi-colon'